.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,14 @@
+[Project Reno Jira Ticket](http://jira.nypl.org/browse/RENO-XXX)
+
+[ResearchNow Jira Ticket](http://jira.nypl.org/browse/SFR-XXX)
+
+## **This PR does the following:**
+-
+
+### UAT Review:
+- [ ] Navigate to [the page in Drupal](https://reno-XXX-nypl.pantheonsite.io/)
+
+### Front End Review:
+- [ ] View [the example in Pattern Lab](https://reno-XXX-nypl.pantheonsite.io/themes/custom/nypl_emulsify/pattern-lab/public)
+- [ ] Compare to [Figma](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Working-Component-Library?node-id=XXXXXX)
+- [ ] Check against the [Metronome documentation](http://themetronome.co/components/xxx/?fresh=true)
```

Comment:
Are permissions going to eventually cause problems here?

Reply:
Everyone on this repo _should_ have permissions. That said I think that checking the Metronome documentation covers the Patternlab example so I'm going to remove that.